src/aks-preview/azext_aks_preview/_consts.py

```diff
@@ -26,3 +26,13 @@
 # refer https://docs.microsoft.com/en-us/rest/api/storageservices/
 # naming-and-referencing-containers--blobs--and-metadata#container-names
 CONST_CONTAINER_NAME_MAX_LENGTH = 63
+
+
+ADDONS = {
+    'http_application_routing': 'httpApplicationRouting',
+    'monitoring': 'omsagent',
+    'virtual-node': 'aciConnector',
+    'azure-policy': 'azurepolicy',
```

Comment:
azurePolicy, should be camel case format.

Reply:
This isn't mine. i just moved it from the custom.py file to the constants file to make it easier to access across various files.